docs/getting_started/Datus_tutorial.zh.md

```
@@ -139,18 +153,18 @@ datus-agent bootstrap-kb \
+datus-cli --datasource california_schools
 /subject
 ```
 
 ![主题树结构](../assets/tutorial_subject_tree.png)
 
-### 步骤 [5/5] 构建 Subagent
+### 步骤 [6/6] 构建 Subagent
 
 教程会自动生成两个 [subagent](../subagent/introduction.zh.md)：
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)